internal/broker/schemas.go

```diff
@@ -0,0 +1,347 @@
+	sapConvergedCloudUpdateSchema := s.SapConvergedCloudSchema(platformRegion, true)
+	freemiumCreateSchema := s.FreeSchema(cp, platformRegion, false)
+	freemiumUpdateSchema := s.FreeSchema(cp, platformRegion, true)
+	trialCreateSchema := s.TrialSchema(false)
+	trialUpdateSchema := s.TrialSchema(true)
+	ownClusterCreateSchema := s.OwnClusterSchema(false)
+	ownClusterUpdateSchema := s.OwnClusterSchema(true)
+	buildRuntimeAWSCreateSchema := s.BuildRuntimeAWSSchema(platformRegion, false)
+	buildRuntimeAWSUpdateSchema := s.BuildRuntimeAWSSchema(platformRegion, true)
+	buildRuntimeGCPCreateSchema := s.BuildRuntimeGcpSchema(platformRegion, false)
+	buildRuntimeGCPUpdateSchema := s.BuildRuntimeGcpSchema(platformRegion, true)
+	buildRuntimeAzureCreateSchema := s.BuildRuntimeAzureSchema(platformRegion, false)
+	buildRuntimeAzureUpdateSchema := s.BuildRuntimeAzureSchema(platformRegion, true)
+	previewCreateSchema := s.PreviewSchema(platformRegion, false)
+	previewUpdateSchema := s.PreviewSchema(platformRegion, true)
+
+	outputPlans := map[string]domain.ServicePlan{
+		AWSPlanID:               defaultServicePlan(AWSPlanID, AWSPlanName, plans, awsCreateSchema, awsUpdateSchema),
+		GCPPlanID:               defaultServicePlan(GCPPlanID, GCPPlanName, plans, gcpCreateSchema, gcpUpdateSchema),
+		AzurePlanID:             defaultServicePlan(AzurePlanID, AzurePlanName, plans, azureCreateSchema, azureUpdateSchema),
+		AzureLitePlanID:         defaultServicePlan(AzureLitePlanID, AzureLitePlanName, plans, azureLiteCreateSchema, azureLiteUpdateSchema),
+		FreemiumPlanID:          defaultServicePlan(FreemiumPlanID, FreemiumPlanName, plans, freemiumCreateSchema, freemiumUpdateSchema),
+		TrialPlanID:             defaultServicePlan(TrialPlanID, TrialPlanName, plans, trialCreateSchema, trialUpdateSchema),
+		OwnClusterPlanID:        defaultServicePlan(OwnClusterPlanID, OwnClusterPlanName, plans, ownClusterCreateSchema, ownClusterUpdateSchema),
+		PreviewPlanID:           defaultServicePlan(PreviewPlanID, PreviewPlanName, plans, previewCreateSchema, previewUpdateSchema),
+		BuildRuntimeAWSPlanID:   defaultServicePlan(BuildRuntimeAWSPlanID, BuildRuntimeAWSPlanName, plans, buildRuntimeAWSCreateSchema, buildRuntimeAWSUpdateSchema),
+		BuildRuntimeGCPPlanID:   defaultServicePlan(BuildRuntimeGCPPlanID, BuildRuntimeGCPPlanName, plans, buildRuntimeGCPCreateSchema, buildRuntimeGCPUpdateSchema),
+		BuildRuntimeAzurePlanID: defaultServicePlan(BuildRuntimeAzurePlanID, BuildRuntimeAzurePlanName, plans, buildRuntimeAzureCreateSchema, buildRuntimeAzureUpdateSchema),
+		SapConvergedCloudPlanID: defaultServicePlan(SapConvergedCloudPlanID, SapConvergedCloudPlanName, plans, sapConvergedCloudCreateSchema, sapConvergedCloudUpdateSchema),
+	}
+	// remove plans if there is no any region defined for given platform region (for example strict list of platform regions for sap converged cloud)
+	for id := range outputPlans {
```

Comment:
Nit picking - wouldn't be easier to add these plans with regions instead of adding all and deleting some?
Subjective again, we need to discuss it briefly (30s).